gui.js

```diff
@@ -4472,22 +4511,6 @@ IDE_Morph.prototype.getURL = function (url) {
     }
 };
 
-IDE_Morph.prototype.getURLsbeOrRelative = function (url) {
```

Comment:
I don't much mind supporting ```file:```, if you've tested this and it works for you I'll pull it next up. Thanks!